### DISCUSSION TITLE
ask for pivot feature

### DISCUSSION THREAD

**Discussion by l1t1** [ClickHouse/ClickHouse#50473] [General]
oracle, duckdb
https://docs.oracle.com/cd/E11882_01/server.112/e41084/statements_10002.htm#SQLRF01702
https://duckdb.org/docs/sql/statements/pivot

**Comment by tbragin** [comment]
It looks like PIVOT/UNPIVOT is possible using other functions https://kb.altinity.com/altinity-kb-queries-and-syntax/pivot-unpivot/

It's reasonable to submit a feature request for adding this other syntax in the main issue tracker though https://github.com/ClickHouse/ClickHouse/issues

**Comment by l1t1** [comment]
thank you
